docs/docs/recipes/manage-users/on-admin-console-page.md

```diff
@@ -5,6 +5,25 @@ sidebar_position: 1
 
 # Manage users on the Admin Console page
 
+In Admin Console, click the **User Management** tab on the left, and you'll see a user table view on the right.
```

Comment:
you need a specific mission or scenario for a recipe, otherwise it'll be a reference

Reply:
updated. PTAL

Reply:
Updated. PTAL

Reply:
Updated. PTAL.